tests/state/test_get_states.py

```diff
@@ -0,0 +1,372 @@
+"""
+Provide tests for command line interface's get a list of states command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    DEV_BRANCH_NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+ADDRESS_WITH_STATE = '0000000000000000000000000000000000000000000000000000000000000000000001'
+
+
+def test_get_states():
+    """
+    Case: get a list of states.
+    Expect: list of states is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        '--node-url',
+        DEV_BRANCH_NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    result_state_addresses = json.loads(result.output).get('result')
+    result_state_addresses = result_state_addresses[0]
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert result_state_addresses.get('address') == ADDRESS_WITH_STATE
+
+
+def test_get_states_with_all_parameters():
+    """
+    Case: get a list of states by account address, starting address, limit, head, reverse.
+    Expect: list of states is returned.
+    """
+    head = 'c892656e3efd5459d782d97cba9cb046a73c7facc12e6849b804a7af7a600949' \
+           '7d554ec1e3f9b191b1cdd512fb5d4b0c256cea3d3bd1475cf1f61a60f74cc4ed'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        '--address',
+        ADDRESS_WITH_STATE,
+        '--start',
+        ADDRESS_WITH_STATE,
+        '--limit',
+        1,
+        '--head',
+        head,
+        '--node-url',
+        DEV_BRANCH_NODE_IP_ADDRESS_FOR_TESTING,
+        '--reverse',
+    ])
+
+    result_state_addresses = json.loads(result.output).get('result')
+    result_state_addresses = result_state_addresses[0]
```

Comment:
`result_state_addresses` > `single_state_address`. If you put one address, a single address is returned. So you should mention it in the code.